info.toml

```diff
@@ -78,13 +78,13 @@ path = "exercises/variables/variables6.rs"
 mode = "compile"
 hint = """
 We know about variables and mutability, but there is another important type of
-variable available; constants.
+variable available: constants.
 Constants are always immutable and they are declared with keyword 'const' rather
 than keyword 'let'.
 Constants types must also always be annotated.
 
-Read more about constants under 'Differences Between Variables and Constants' in the book's section 'Variables and Mutability':
-https://doc.rust-lang.org/book/ch03-01-variables-and-mutability.html#differences-between-variables-and-constants
+Read more about constants and the differences between variables and constants under 'Constants' in the book's section 'Variables and Mutability':
```

Comment:
Open to other wordings here. I think it's valuable to clearly mention the differences between variables and constants since the book does as well.